pom.xml

```diff
@@ -122,7 +122,7 @@
 							<goal>generate</goal>
 						</goals>
 						<configuration>
-							<inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-ecommerce-payment-instruments-service/test-helm-deploy/api-config-mock/openapi.json</inputSpec>
+							<inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-ecommerce-payment-instruments-service/main/api-config-mock/openapi.json</inputSpec>
```

Comment:
you can use swagger of api-config repo. after this PR https://github.com/pagopa/pagopa-api-config/pull/125

Reply:
Great, solved with [b87b662](https://github.com/pagopa/pagopa-ecommerce-payment-instruments-service/pull/9/commits/b87b66264638b1bcdcb27d427c2992de9b8e36d8) 👍